src/gdft.c

```diff
@@ -431,6 +431,68 @@ gdTcl_UtfToUniChar (char *str, Tcl_UniChar * chPtr)
 	return 1;
 }
 
+#ifdef HAVE_LIBRAQM
+#include <raqm.h>
+#else
+typedef struct {
+	unsigned int index;
+		int x_advance;
```

Comment:
indentation is broken here

also, you use int, but you store values that are FT_Pos (signed long), so perhaps these should all be FT_Pos ?